docs/docsite/rst/dev_guide/developing_plugins.rst

```diff
@@ -513,6 +511,23 @@ The parameters are:
 
 This ``get_vars`` method just needs to return a dictionary structure with the variables.
 
+.. code-block:: python
+
+    from ansible.module_utils.common.text.converters import to_text
```

Comment:
I don't think this is a very useful example since you could just return "some test" directly. I don't think this should be the place to document every common data manipulator a plugin might import.